portingdb/templates/mispackaged.html

```diff
@@ -0,0 +1,56 @@
+{% extends "_base.html" %}
+
+{% block titlecontent -%}
+    Mispackaged – {{ super() }}
+{%- endblock titlecontent %}
+
+{% block style %}
+{{ super() }}
+
+body {
+  /*font: 10px sans-serif;*/
+}
+
+span.halfwidth {
+    width: 50%; display: inline-block;
+}
+
+{% endblock style %}
+
+{% block bodycontent %}
+
+<div class="container">
+    <div class="col-md-12">
+        <h1>Mispackaged packges</h1>
+        <p>
+            {% if not requested %}
+                You can see <i>all packages</i> or <a href="{{ url_for('mispackaged', requested=1) }}">packages where the maintainer requested a patch</a>.
+            {% else %}
+                You can see <a href="{{ url_for('mispackaged') }}">all packages</a> or <i>packages where the maintainer requested a patch</i>.
+            {% endif %}
+        </p>
+        <ul class="simple-pkg-list">
```

Comment:
Hm, I don't know why but I was pretty sure you couldn't. Probably IE5.5 for which I started writing HTML didn't display it correctly and thus my memory cemented this as impossible. :)